test/spec/spec.controller.js

```diff
@@ -20,6 +20,11 @@ describe('lib/base-controller', () => {
     sinon.stub(BaseController.prototype, 'locals').returns({foo: 'bar'});
   });
 
+  it('sets the correct error class to the instance', () => {
+    controller = new Controller({});
+    controller.Error.should.be.equal(require('../../lib/error'));
```

Comment:
It's effectively doing voluntarily what ES module imports do as part of the spec. `import` statements are automatically hoisted to the top level to be statically analysable.